network/protocol.go

```diff
@@ -0,0 +1,14 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022 Datadog, Inc.
+
+package network
+
+type Protocol string
+
+const (
+	TCP Protocol = "tcp"
```

Comment:
What about an `Undefined Protocol = ""` to make the function call a little bit more obvious?